lib/find_retryables.ts

```diff
@@ -119,45 +105,42 @@ const main = async (
       { fromBlock, toBlock },
       parentChainProvider
     )
-    // Create a set to store unique transaction hashes
+
     const uniqueTxHashes = new Set<string>()
 
-    // Iterate through inboxDeliveredLogs and add unique transaction hashes to the set
     for (let inboxDeliveredLog of inboxDeliveredLogs) {
       if (inboxDeliveredLog.data.length === 706) continue // depositETH bypass
```

Comment:
I'd need @DZGoldman to confirm, but I think this is not valid anymore. On my tests, it looks like the script is picking up ethdeposits as well, which makes it very slow to finish.

An alternative could be to search by MessageDelivered events from the Bridge contract (instead of InboxMessageDelivered events from the Inbox), and filter the ones that have a "kind" field different than 9 (SubmitRetryableTx).